blueprints.yaml

```diff
@@ -20,12 +20,30 @@ form:
   validation: loose
   fields:
     enabled:
-      type: toggle
       label: PLUGIN_ADMIN.PLUGIN_STATUS
+      type: toggle
       highlight: 1
       default: 0
       options:
-        1: PLUGIN_ADMIN.ENABLED
         0: PLUGIN_ADMIN.DISABLED
+        1: PLUGIN_ADMIN.ENABLED
       validate:
         type: bool
+
+    processor:
+      label: Processor
```

Comment:
Both labels (processor and formats) would need to have proper translation strings.
 Something like:
```
PLUGIN_TRANSLATE_DATE:
  fields:
    processor:
      label: Processor
    formats:
      label: Formats
      placeholder:
        key: Language
        value: Format
  format:
    F: ...
    M: ...
    l: ...
    D: ...
```

I guess it would require adjusting [the filter class](https://github.com/Karmalakas/grav-plugin-translate-date/blob/main/classes/Twig/FilterTd.php#L198) too.

Also would require adding info about at least formats field to the README

Reply:
I know. I just saw your notice in the readme and the issue tracker and made a minimal commit.

I think it will work out of the box as the config is stored in the exact same format as before.